client/state/themes/actions/add-external-managed-theme-to-cart.tsx

```diff
@@ -127,8 +127,12 @@ export function addExternalManagedThemeToCart( themeId: string, siteId: number )
 		const { origin = 'https://wordpress.com' } =
 			typeof window !== 'undefined' ? window.location : {};
 
+		const redirectTo = addQueryArgs( `${ origin }/theme/${ themeId }/${ siteSlug }`, {
+			...( ! isSiteEligibleForManagedExternalThemes ? { 'sync-active-theme': true } : {} ),
```

Comment:
Might it be worth using the string `'true'` here given that we're expecting that as the incoming value? The transform from `true` to `'true'` feels closely tied to the underlying implementation, when that could easily be understood to represent `&sync-active-theme=1` or just `&sync-active-theme` with no value.
```suggestion
			...( ! isSiteEligibleForManagedExternalThemes ? { 'sync-active-theme': 'true' } : {} ),
```